sk0-back/src/users/controllers/users.controller.ts

```diff
@@ -18,15 +17,9 @@ export class UsersController {
   @UsePipes(ValidationPipe)
   async loginUser(@Body() loginUserDto: LoginUserDto) {
     
-    const existingUser = await this.usersService.findOneByEmail(
-        loginUserDto.email,
-    );
+    const existingUser = await this.usersService.validateUser(loginUserDto.email, loginUserDto.password);
 
-    if (!existingUser) {
-      throw new NotFoundException('User not found!');
-    }
-
-    if (existingUser.password === loginUserDto.password) {
+    if (existingUser) {
       return { message: 'Login successful' };
     } else {
       throw new UnauthorizedException('Invalid credentials');
```

Comment:
I think a simple authservice call would be enough:
```typescript
  async loginUser(@Body() loginUserDto: LoginUserDto) {
    return authService.login(loginUserDto);
  }
```
and you can inject the AuthService in the constructor